### PR TITLE
Replace broken link with working link

### PR DESCRIPTION
**Pull request by markcmurphy**
# [DEVDOCS-2433](https://jira.bigcommerce.com/browse/DEVDOCS-2433)

## What changed?
Broken URLs were updated with working links to content

### CODE REVIEW

File: docs/api-docs/store-management/shipping/shipping-provider-api.md [hunk 1]
```diff
@@ -475,34 +477,38 @@ Only one registered shipping carrier may be associated with an app. This [regist
...
 ### Endpoints
+
 - [Shipping Provider](/api-reference/store-management/shipping-provider-api)
 - [Shipping Zones](/api-reference/store-management/shipping-api/shipping-zones)
 - [Shipping Methods](/api-reference/store-management/shipping-api/shipping-method)
 - [Shipping Carriers](/api-reference/store-management/shipping-api/shipping-carrier)
 
 ### Webhooks
+
 - [Shipping](/api-docs/getting-started/webhooks/webhook-events#webhook-events_shipment)
```
Comment (traci707) [inline docs/api-docs/store-management/shipping/shipping-provider-api.md:514]: - [Webhooks](/api-docs/getting-started/webhooks/webhook-events#webhook-events_shipment)

File: docs/api-docs/store-management/shipping/shipping-provider-api.md [hunk 2]
```diff
@@ -475,34 +477,38 @@ Only one registered shipping carrier may be associated with an app. This [regist
...
 
 ## Related resources
 
 ### Articles
+
 - [App Store Approval Requirements](https://developer.bigcommerce.com/api-docs/partner/app-store-approval-requirements)
 
 ### Endpoints
+
 - [Shipping Provider](/api-reference/store-management/shipping-provider-api)
```
Comment (traci707) [inline docs/api-docs/store-management/shipping/shipping-provider-api.md:507]: - [Shipping Providers](/api-reference/store-management/shipping-provider-api)

File: docs/api-docs/store-management/shipping/shipping-provider-api.md [hunk 3]
```diff
@@ -475,34 +477,38 @@ Only one registered shipping carrier may be associated with an app. This [regist
...
+| Quote URL                       | A URL for a resource of the shipping carrier that accepts quote requests and responds with shipping quotes. For more on the Quote URL, see a typical app workflow below.                                                                                                                                                                                                                                                                                                                                                        |
+| Single Carrier vs Multi Carrier | A single carrier app will offer only one shipping provider. A multi carrier app will aggregate multiple shipping carriers in one app.                                                                                                                                                                                                                                                                                                                                                                                           |
```
Comment (traci707) [inline docs/api-docs/store-management/shipping/shipping-provider-api.md:484]: | Single Carrier vs. Multi Carrier | A single carrier app will offer only one shipping provider. A multi carrier app will aggregate multiple shipping carriers in one app.

File: docs/api-docs/store-management/shipping/shipping-provider-api.md [hunk 4]
```diff
@@ -475,34 +477,38 @@ Only one registered shipping carrier may be associated with an app. This [regist
...
+| Countries Available             | A list of countries where you can use the shipping carrier. The default behavior is that the carrier is available for every shipping origin. In most cases, this list should be as broad as possible. For example, if your carrier operates worldwide, make it available worldwide. The countries can be limited further than what the shipping carrier has provided. If the service is worldwide, then leave this field blank to specify that it is worldwide. Specifying the use of the shipping carrier is an optional step. |
```
Comment (traci707) [inline docs/api-docs/store-management/shipping/shipping-provider-api.md:485]: | Countries Available             | A list of countries where you can use the shipping carrier. The default behavior is that the carrier is available for every shipping origin. In most cases, this list should be as broad as possible. For example, if your carrier operates worldwide, make it available worldwide. You can limit the countries further than what the shipping carrier has provided. If the service is worldwide, then leave this field blank to specify that it is worldwide. Specifying the use of the shipping carrier is an optional step. |

File: docs/api-docs/store-management/shipping/shipping-provider-api.md [hunk 5]
```diff
@@ -21,17 +22,17 @@
 
 Shipping service providers wishing to offer shipping services and rates to BigCommerce merchants and shoppers can implement BigCommerce Shipping Provider endpoints. Once they implement and accept their service into BigCommerce's shipping Carrier Registry, merchants will have access to enable and configure the service through their BigCommerce control panel. Once enabled on a store, BigCommerce will automatically retrieve the service options and rates using the provider's endpoints and display them to merchants in the store's control panel and to shoppers on the storefront.
 
-
 Shipping Provider endpoints can also be used by merchants to retrieve rates from custom shipping tables or an in-house shipping rate calculation service.
```
Comment (traci707) [inline docs/api-docs/store-management/shipping/shipping-provider-api.md:25]: Merchants can also use shipping provider endpoints to retrieve rates from custom shipping tables or an in-house shipping rate calculation service.

**Review by traci707 — CHANGES_REQUESTED** [review]
While in the file, I found a few instances of passive voice which I changed.  I also made an editorial change (vs.) and updated an endpoint name under related resources.

**Review by traci707 — APPROVED** [review]

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**